### PR TITLE
"Reset" button in UI

### PR DESCRIPTION
**Pull request by samsondav**
"Reset" button in admin UI allows configuring key/chain.
Screenshot 2022-09-12 at 2 13 43 PM

### CODE REVIEW

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 1]
```diff
@@ -1,48 +1,347 @@
...
 import { TimeAgo } from 'src/components/TimeAgo'
 
+const styles = (theme: Theme) =>
+  createStyles({
+    container: {
+      backgroundColor: theme.palette.common.white,
+      padding: theme.spacing.unit * 5,
+      paddingBottom: 0,
+    },
+    mainRow: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:41]: Can be deleted, not used

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 2]
```diff
@@ -1,48 +1,347 @@
...
+      marginBottom: theme.spacing.unit * 2,
+    },
+    actions: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:44]: Ditto

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 3]
```diff
@@ -1,48 +1,347 @@
...
+      textAlign: 'right',
+    },
+    regionalNavButton: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:47]: Ditto

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 4]
```diff
@@ -1,48 +1,347 @@
...
+      marginLeft: theme.spacing.unit,
+      marginRight: theme.spacing.unit,
+    },
+    horizontalNav: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:51]: Ditto

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 5]
```diff
@@ -1,48 +1,347 @@
...
+      paddingBottom: 0,
+    },
+    horizontalNavItem: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:54]: Ditto

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 6]
```diff
@@ -1,48 +1,347 @@
...
+      display: 'inline',
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    horizontalNavLink: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:59]: Ditto

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 7]
```diff
@@ -1,48 +1,347 @@
...
+      padding: `${theme.spacing.unit * 4}px ${theme.spacing.unit * 4}px`,
+      textDecoration: 'none',
+      display: 'inline-block',
+      borderBottom: 'solid 1px',
+      borderBottomColor: theme.palette.common.white,
+      '&:hover': {
+        borderBottomColor: theme.palette.primary.main,
+      },
+    },
+    activeNavLink: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:69]: Ditto

File: operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx [hunk 8]
```diff
@@ -1,48 +1,347 @@
...
+    closeButton: {
+      marginRight: theme.spacing.unit * 3,
+      marginTop: theme.spacing.unit * 3,
+    },
+    infoText: {
+      fontSize: theme.spacing.unit * 2,
+      fontWeight: 450,
+      marginLeft: theme.spacing.unit * 6,
+    },
+    modalTextarea: {
```
Comment (Alejandro-Larumbe) [inline operator_ui/src/screens/KeyManagement/EVMAccountRow.tsx:107]: Ditto

**Review by Alejandro-Larumbe — DISMISSED** [review]
Left some small non-blocking comments. Otherwise LGTM

File: core/services/keystore/eth.go [hunk 9]
```diff
@@ -109,6 +109,7 @@ func (ks *eth) Create(chainIDs ...*big.Int) (ethkey.KeyV2, error) {
 	if err == nil {
 		ks.notify()
 	}
+	ks.logger.Infow(fmt.Sprintf("Created EVM key with ID %s", key.Address.Hex()), "address", key.Address.Hex(), "evmChainIDs", chainIDs)
```
Comment (jmank88) [inline core/services/keystore/eth.go:112]: What format does `chainIDs` get logged in? JSON array?
Might we later get annoyed by the plural key name `evmChainIDs` when trying to filter logs by keys and `evmChainID` doesn't match this?

Comment (samsondav) [reply core/services/keystore/eth.go:112]: mm, good point

Comment (samsondav) [reply core/services/keystore/eth.go:112]: Yes, JSON array.

I think the plural is ok.

**Pull request merged**